src/com/Jacksonnn/DCCore/RandomTP/RandomTP.java

```diff
@@ -21,7 +21,7 @@ public boolean onCommand(CommandSender sender, Command command, String s, String
                     sender.sendMessage(GeneralMethods.errorPrefix);
                     return true;
                 }
-                World world = player.getWorld();
+                World world = world;
```

Comment:
Why did you set it to an uninitialized version of itself? That sets the value to itself, but since it doesn't have a value, to begin with, it's null.

We use player.getWorld() to mark the world they'll teleport in.

Reply:
Is there a certain goal of editing RandomTP that you're trying to achieve?

Reply:
Essentially, I would like the RTP to be accessible anywhere and to be sent to the regular survival world.
The banish command was originally used to circumvent the dos player. This command along with /woosh, /piss and multiple others show up within the tab complete in the command line. 

Since the hacker has given up we have upgraded security, encryption, and we have our players protected by proxy. 

at this point having these commands accessible to regular players is a con that outways the pros.